### ISSUE TITLE
Use a nicer deliminator for replies

### ISSUE THREAD

**Issue by claell** [ecederstrand/exchangelib#1220]
**Is your feature request related to a problem? Please describe.**
The current deliminator for the original content and the new message in replies is not that nice looking. Compare the one from exchangelib to the one in Outlook (365):

exchangelib:
![grafik](https://github.com/ecederstrand/exchangelib/assets/6171135/a0ba8eed-d966-4076-b4c4-c8acdfc9cc12)

Outlook:
![grafik](https://github.com/ecederstrand/exchangelib/assets/6171135/35332bdb-44cf-4c95-b466-b02fc7b9355e)

**Describe the solution you'd like**
Possibly, it is easy to replicate the Outlook deliminator and use that.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by ecederstrand** [comment]
We don't specify any delimiter in exchangelib. It's possible that the differenct is caused by Outlook sending emails in HTML format where exchangelib defaults to text unless you use `HTMLBody` explicitly.

**Comment by claell** [comment]
I already used `HTMLBody`, as the normal text delimiter is even "worse".

Possibly, Outlook is setting a different delimiter, explicitly, then.

**Comment by ecederstrand** [comment]
Sounds likely. Try looking at the raw HTML of a "nice" email to find out how they define the delimiter.

**Comment by claell** [comment]
I think it's in between here:

```
<p class="MsoNormal"><span style="mso-bookmark:_MailEndCompose"><span style="mso-bookmark:_MailAutoSig"><span style="mso-ascii-font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:&quot;Times New Roman&quot;;mso-fareast-theme-font:minor-fareast;mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-no-proof:yes"><o:p>&nbsp;</o:p></span></span></span></p>
</div>
<span style="mso-bookmark:_MailAutoSig"></span>
<p class="MsoNormal"><span style="mso-bookmark:_MailEndCompose"><span style="mso-ascii-font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-fareast-language:EN-US"><o:p>&nbsp;</o:p></span></span></p>
<span style="mso-bookmark:_MailEndCompose"></span>
<div>
<div style="border:none;border-top:solid #E1E1E1 1.0pt;padding:3.0pt 0cm 0cm 0cm">
<p class="MsoNormal"><a name="_MailOriginal"><b><span style="mso-fareast-font-family:&quot;Times New Roman&quot;">Von:</span></b></a><span style="mso-bookmark:_MailOriginal"><span style="mso-fareast-font-family:&quot;Times New Roman&quot;"> Ellsel, Claudius Matts &lt;mail@address.com&gt;
<br>
```

**Comment by claell** [comment]
Exchangelib:

```
<span class="GramE">Test Antwort</span> Viele Grüße Claudius Ellsel <o:p></o:p></span></p>
<div class="MsoNormal" align="center" style="text-align:center"><span style="mso-fareast-font-family:&quot;Times New Roman&quot;">
<hr size="2" width="98%" align="center">
</span></div>
<div id="divRplyFwdMsg">
<p class="MsoNormal"><b><span lang="EN-US" style="mso-fareast-font-family:&quot;Times New Roman&quot;;color:black;mso-ansi-language:EN-US">From:</span></b><span lang="EN-US" style="mso-fareast-font-family:&quot;Times New Roman&quot;;color:black;mso-ansi-language:EN-US"> Ellsel,
 Claudius Matts<br>
```

**Comment by claell** [comment]
Am I right that this is also handled on server side, and you only send a reply item through EWS that doesn't allow adjusting this?

Then, this issue can be closed for the same reason as in #1219.

**Comment by ecederstrand** [comment]
Yes, that's correct. As in the other issue, you can always send a full `Message` item and format the rely exactly as you want.
